lib/publiccloud/instance.pm

```diff
@@ -372,7 +372,7 @@ Return:
 sub wait_for_ssh {
     my ($self, %args) = @_;
     # Input parameters, see description in above head2 - Parameters section:
-    $args{timeout} //= 600;
+    $args{timeout} = get_var('PUBLIC_CLOUD_SSH_TIMEOUT', $args{timeout} // 600);
```

Comment:
```suggestion
    $args{timeout} //= get_var('PUBLIC_CLOUD_SSH_TIMEOUT', 600);
```

Reply:
No Mike sorry :smile: I just changed that code that was quite as you're proposing now, see the thread ...

Reply:
I hope that link will clarify the last change